### ISSUE TITLE
[FR] Separate load and run calls in the runtime.

### ISSUE THREAD

**Issue by paullewis** [google/labs-prototypes#268]
In the Main Thread Runtime (and the other runtimes, I assume), we [load and start the board](https://github.com/google/labs-prototypes/blob/main/seeds/breadboard-web/src/main-thread-runtime.ts#L53-L64) at the same time. I was thinking it'd be nice to load the board and show its graph without necessarily starting up the board, then we can also do things like resetting the board without reloading it.

Wdyt, @dglazkov?

**Comment by dglazkov** [comment]
Sure thing! FWIW, you should be able to accomplish this by waiting for "load" run result, and not giving control back to async gen until you're ready.

**Comment by paullewis** [comment]
Yeah, that's how I'm handling it right now. But maybe this also ties into #269 in that having a separate load phase might allow us to do more interesting things in terms of errors & validation. I mean, it's doable in the current approach, it just feels like it might be cleaner to have separate `load` and `run` calls.

**Comment by dglazkov** [comment]
I am starting to imagine something like maybe an inspector package. Something that doesn't run the graph, but loads it and allows poking at it to get useful stuff.

What would be the stuff that you would need for debugger? Let's see if we can spec that.
